### DISCUSSION TITLE
Snapping für WMS-Dienste

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#139] [General]
Wäre es möglich, dass man das Snapping-Tool auch für WMS-Dienste implementieren könnte? z.B. würden wir das für den BEV-WMS Dienst (tagesaktuelle DKM) benötigen.

**Comment by gis-eni** [comment]
WMS Dienste liefern "nur" Bilder.  Für ein Snapping sind aber Vektordaten notwendig. Also nein, technisch nicht möglich.

Falls jetzt die Frage kommt... ;)
Nein, es ist auch für Vektortiles nicht möglich. Das wäre zwar Vektoren drin, allerdings optimiert auf die Bildschirm (Pixel) Koordinaten. Darauf sollte man natürlich auch nicht snappen, weil das je nach aktuellem Maßstab auch weit von der Wahrheit weg ist.

Snapping funktioniert nur auf Dienste, die die tatsächlichen Koordinaten für die Features liefern, zB AGS Map/Feature Services, gView Dienste, WFS

**Reply by thomaskofler** [reply]
ok passt, danke für die Info
